src/worker/worker.py

```diff
@@ -188,36 +189,41 @@ def validate_chunks(chunked_data, chunk_validation_url):
 
 def chunk_data_exact(data_chunks, chunk_size, chunk_overlap):
```

Comment:
Can you add a comment above this method explaining what its doing? is it converting all the characters to tokens then creating a token array and sliding through the token array?

Reply:
Added the requested comment